src/main/java/org/candlepin/subscriptions/marketplace/MarketplaceSubscriptionIdCollector.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2021 Red Hat, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * Red Hat trademarks are not licensed under GPLv3. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+package org.candlepin.subscriptions.marketplace;
+
+import org.candlepin.subscriptions.marketplace.api.resources.MarketplaceApi;
+import org.candlepin.subscriptions.tally.UsageCalculation.Key;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+/**
+ * Class responsible for communicating with the Marketplace API and fetching the subscription ID.
+ */
+@Component
+public class MarketplaceSubscriptionIdCollector {
+    public static final String DUMMY = "DUMMY";
+    private final MarketplaceApi marketplaceApi;
+
+    @Autowired
+    public MarketplaceSubscriptionIdCollector(MarketplaceApi marketplaceApi) {
+        this.marketplaceApi = marketplaceApi;
+    }
+
+    public String fetchSubscriptionId(String orgId, Key usageKey) {
+        return DUMMY;
```

Comment:
@awood, can you read from an env var for short term? Even something with `System.getenv()` would be fine, since we don't plan to do long term (though I'm fine if you want to go ahead and place in our config file per norm).